### DISCUSSION TITLE
Ask for wake/start command

### DISCUSSION THREAD

**Discussion by Phoenix-DH** [AlexGustafsson/homebridge-wol#232] [Questions]
Hey @AlexGustafsson,

I used your plugin with pleasure for years on my homebridge.
I powered on my NAS and shut it down with no issues and ko need to go to the roof.
Last time when homebridge crashed I decided to move to HA.

There i make my first steps.
Want to also have the functionality to poweron and off the NAS.
Shut down I get managed, but powering on, I have no change to get it managed.

Would it be possible to provide the code/command which you are using here?

Thanks a lot!

**Comment by AlexGustafsson** [comment]
If by HA you mean Home Assistant, then it's worth mentioning that plugins there are typically written in Python. This is a plugin for Homebridge, written in TypeScript / JavaScript. For starting things over the network we use Wake On LAN via the following package: https://www.npmjs.com/package/wake_on_lan. It has some example code.
